app/views/play/cinematic/dialogSystem.js

```diff
@@ -0,0 +1,167 @@
+import anime from 'animejs/lib/anime.es.js'
```

Comment:
This file can be ignored as an example stub for the Dialog System to grow from.
None of its functionality is required in this PR.

I can delete this system entirely if that's advantageous.